src/sentry/workflow_engine/handlers/action/notification/issue_alert.py

```diff
@@ -298,26 +302,25 @@ def get_target_display(cls, action: Action, mapping: ActionFieldMapping) -> dict
     def get_target_identifier(
         cls, action: Action, mapping: ActionFieldMapping, organization_id: int
     ) -> dict[str, Any]:
+        target_id = action.config.get("target_identifier")
+        target_type = action.config.get("target_type")
+
         # this would be when the target_type is IssueOwners
-        if action.target_identifier is None:
-            if action.target_type != ActionTarget.ISSUE_OWNERS.value:
+        if target_id is None:
+            if target_type != ActionTarget.ISSUE_OWNERS.value:
                 raise ValueError(
-                    f"No target identifier found for {action.type} action {action.id}, target_type: {action.target_type}"
+                    f"No target identifier found for {action.type} action {action.id}, target_type: {target_type}"
                 )
             return {}
         else:
-            return {
-                mapping[
-                    ActionFieldMappingKeys.TARGET_IDENTIFIER_KEY.value
-                ]: action.target_identifier
-            }
+            return {mapping[ActionFieldMappingKeys.TARGET_IDENTIFIER_KEY.value]: target_id}
 
     @classmethod
     def get_additional_fields(cls, action: Action, mapping: ActionFieldMapping) -> dict[str, Any]:
-        if action.target_type is None:
-            raise ValueError(f"No target type found for {action.type} action {action.id}")
+        target_type = action.config.get("target_type")
```

Comment:
should this still be cast to the `ActionTarget` enum?

Reply:
🤔 we probs don't need the if none statement if we use the `ActionTarget` too -- that raises a value error if it fails already.